### DISCUSSION TITLE
RustDesk ID/Relay Server Selection

### DISCUSSION THREAD

**Discussion by abdullah-erturk** [rustdesk/rustdesk#6152] [General]
We have installed a rustdesk server in our institution and it works fine within the corporate network, but sometimes it is necessary to connect to other computers outside the corporate network. We do this by deleting the server information through Rustdesk ID/Relay settings, but this information must be entered again.

It would be great if there were 2 ID/Relay settings on the Rustdesk program and the user could choose the server side or the server running over the internet.

I hope I could explain what I mean.

![rustdesk](https://github.com/rustdesk/rustdesk/assets/54866587/a7e2b185-18ad-401a-8a90-a67090783e31)

**Comment by rustdesk** [comment]
close as many duplicate
